File: fine_tuning/results/test_output_files/question-59-pass-1.py
from math import pow, ceil


def solve(n):
    for (i, j) in zip(range(1, ceil(pow(n, 0.25))), range(ceil(pow(n, 0.25)), 0, -1)):
        if (pow(i, 4) - pow(j, 4) == n):
            print(i, j)
            break


if __name__ == "__main__":
    solve(16)

File: results/feedback_pipeline/cpp_results/cpp_files_before_feedback/question-59-pass-1.cpp
#include <bits/stdc++.h> 
 using namespace std ;
 void solve ( int n ) {
int a, b;
for (int i = 1; i <= n; i++) {
    for (int j = i + 1; j <= n; j++) {
        if (pow(i - j, 4) == n) {
            a = i;
            b = j;
            break;
        }
    }
    if (a!= 0 && b!= 0) {
        break;
    }
}
cout << a << " " << b << endl;